core/cat/factory/llm.py

```diff
@@ -139,8 +139,8 @@ class LLMAzureOpenAIConfig(LLMSettings):
     # Current supported versions 2022-12-01, 2023-03-15-preview, 2023-05-15
     # Don't mix api versions: https://github.com/hwchase17/langchain/issues/4775
     api_version: str = "2023-05-15"
-    deployment_name: str = "text-davinci-003"
-    model_name: str = "text-davinci-003"  # Use only completion models !
+    deployment_name: str = "gpt-35-turbo"
```

Comment:
The correct value here is `gpt-35-turbo-instruct`. It is not released yet, but it is "coming soon", as announced here:

https://techcommunity.microsoft.com/t5/azure-ai-services-blog/announcing-updates-to-azure-openai-service-models/ba-p/3866757